### DISCUSSION TITLE
room-assistant terminates Airplay music streaming?

### DISCUSSION THREAD

**Discussion by Pojay-hcz** [mKeRix/room-assistant#343] [Q&A]
good morning to everyone,
everytime room-assistant is active my audio stream via Airplay is terminated soon after starting it. The stream may last between 10 sec to 1 min before it stopps. Vice versa if RA is inactive the audio stream remains stable and flawless.
Audio streams via Bluetooth connection between paired devices do not seem to be affected.
I would appreciate if you could share your experiences.
Thank you very much, Claus

My HW/SW environment is:
room assistant 2.12.0 installed on Raspberry Pi 4 (installation according guideline room-assistant.io)
HomeAssistant 0.117.5/Mosquitto 5.1 installed in Virtualbox 61. installed in Windows 10 Pro Version 2004
iPhone 8 with iOS 14.2 (as device tracker for presence detection).

**Comment by mKeRix** [comment]
AirPlay doesn't use Bluetooth afaik, so I'm not sure why this would happen. I also have not observed this behavior in my setup yet, although the conditions are slightly different as I'm not tracking my personal iPhone atm, only my Watch. I can try to run some experiments surrounding this soon-ish.

**Reply by Pojay-hcz** [reply]
Before I had the same opinion like you. That is why, this was one of my first tests:
1. room-assistant remains active
2. turn off BT interface of my iPhone (it should not have an effect to the Airplay stream, but room-assistant should not have BT contact to the iPhone)
Unexpectedly, Airplay stopped immediately, new text appeared in iPhone settings, saying “AirDrop, Airplay, Where is? and location services are using BT”.
In other words: these services will not work without BT.
